rootex/core/renderer/rendering_device.cpp

```diff
@@ -148,6 +156,18 @@ void RenderingDevice::initialize(HWND hWnd, int width, int height)
 		dsDesc.DepthEnable = TRUE;
 		dsDesc.DepthFunc = D3D11_COMPARISON_LESS;
 		dsDesc.DepthWriteMask = D3D11_DEPTH_WRITE_MASK_ALL;
+		dsDesc.StencilEnable = TRUE;
+		dsDesc.StencilReadMask = 0xff;
+		dsDesc.StencilWriteMask = 0xff;
+		dsDesc.FrontFace.StencilDepthFailOp = D3D11_STENCIL_OP_KEEP;
+		dsDesc.FrontFace.StencilPassOp = D3D11_STENCIL_OP_REPLACE;
+		dsDesc.FrontFace.StencilFailOp = D3D11_STENCIL_OP_REPLACE;
+		dsDesc.FrontFace.StencilFunc = D3D11_COMPARISON_ALWAYS;
+		dsDesc.BackFace.StencilDepthFailOp = D3D11_STENCIL_OP_KEEP;
+		dsDesc.BackFace.StencilPassOp = D3D11_STENCIL_OP_REPLACE;
+		dsDesc.BackFace.StencilFailOp = D3D11_STENCIL_OP_REPLACE;
+		dsDesc.BackFace.StencilFunc = D3D11_COMPARISON_ALWAYS;
```

Comment:
keep this as `D3D11_COMPARISON_NEVER` since we don't really care for the backfacing triangles

Reply:
No ModelComponent gets rendered if we do this.